CollapseLauncher/Classes/Helper/Database/DBHandler.cs

```diff
@@ -10,355 +11,412 @@
 // ReSharper disable CommentTypo
 
 #nullable enable
-namespace CollapseLauncher.Helper.Database
+namespace CollapseLauncher.Helper.Database;
+
+internal static class DbHandler
 {
-    internal static class DbHandler
-    {
-        #region Config Properties
+    #region Config Properties
 
-        private static bool? _enabled;
-        public static bool IsEnabled
+    private static bool? _enabled;
+    public static bool IsEnabled
+    {
+        get
         {
-            get
-            {
-                if (_enabled != null) return (bool)_enabled;
-                var c = DbConfig.DbEnabled;
-                _enabled = c;
-                return c;
-            }
-            set
-            {
-                _enabled           = value;
-                DbConfig.DbEnabled = value;
+            if (_enabled != null) return (bool)_enabled;
+            var c = DbConfig.DbEnabled;
+            _enabled = c;
+            return c;
+        }
+        set
+        {
+            _enabled           = value;
+            DbConfig.DbEnabled = value;
 
-                _isFirstInit = true; // Force first init
-                if (!value) Dispose(); // Dispose instance if user disabled database function globally
-            }
+            _isFirstInit = true; // Force first init
+            if (!value) Dispose(); // Dispose instance if user disabled database function globally
         }
+    }
```

Comment:
Really not necessary but can we use ``field`` keyword instead of manual ``_enabled`` field?
Also, add property attribute with ``AllowNull`` and ``MaybeNull`` for ``field`` keyword, like:

```CSharp
[field: AllowNull, MaybeNull]
public static bool IsEnabled
{
    get
    {
        if (field != null)
        {
            return field;
        }
        // Do something else
    }
    set
    {
        field = value;
        // Do something else
    }
}
```

Reply:
Implemented in https://github.com/CollapseLauncher/Collapse/pull/791/commits/d9de2cac5a3ff89372844e29a4f27df6e7421f42 and https://github.com/CollapseLauncher/Collapse/pull/791/commits/1c1fdd3563b1e484c6fd880ff24f903e5eec044b